AppSettingsManagerUI/src/app/services/auth/auth.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { BffService } from '../bff/bff.service';
+import { User } from '../bff/models/User';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthService {
+  private loggedIn = false;
+
+  private username: string = ``;
+  private password: string = ``;
+
+  constructor(private bffService: BffService) { }
+
+  async login(username: string, password: string): Promise<boolean> {
+    this.loggedIn = await this.bffService.authenticateUser(username, password);
+    this.username = username;
+    this.password = password;
+    return this.loggedIn;
+  }
```

Comment:
For storing passwords, we will definitely need a hashing algorithm for added security. This might be one of our stretch goals, but it is a very realistic practice that we should learn to use and adopt.